cmd/rofl/mgmt.go

```diff
@@ -463,6 +479,37 @@ var (
 		},
 	}
 
+	artifactsUpgradeCmd = &cobra.Command{
+		Use:   "artifacts-upgrade",
```

Comment:
We could simply use `upgrade` if we'll be updating some other components as well in the future (rofl.yaml file format, policy).

```suggestion
		Use:   "upgrade",
```

Reply:
My only concern is that we also have "update" and this sounds so similar?